apps/ex_nvr/lib/ex_nvr/devices.ex

```diff
@@ -43,6 +43,7 @@ defmodule ExNVR.Devices do
   def list(params \\ %{}), do: Repo.all(Device.filter(params) |> order_by([d], d.inserted_at))
 
   @spec get(binary()) :: Device.t() | nil
+  def get(nil), do: nil
```

Comment:
Is this still used ?

Reply:
Yes, required in the event controller
https://github.com/evercam/ex_nvr/blob/39bcb5b50c5933f91dacbfad3de77c70163d7a46/apps/ex_nvr_web/lib/ex_nvr_web/controllers/api/event_controller.ex#L15